pyiron_base/generic/hdfio.py

```diff
@@ -291,40 +291,60 @@ def copy_to(self, destination, file_name=None, maintain_name=True):
             FileHDFio: FileHDFio object pointing to a file which now contains the same content as file of the current
                        FileHDFio object.
         """
+        def _internal_copy(source, source_path, target, target_path, maintain_flag):
+            """
+            Internal function to copy content of one HDF5 file to another or copy a group within the same HDF5 file. 
+            
+            Args:
+                source (h5py.File): HDF5 File object 
+                source_path (str): Path inside the source HDF5 file 
+                target (h5py.File): HDF5 File object
+                target_path (str): Path inside the target HDF5 file 
+                maintain_flag (bool): Maintain the same group name 
+            """
+            if maintain_flag:
+                try:
+                    target.create_group(target_path)
+                except ValueError:
+                    pass  # In case the copy_to() function failed previously and the group already exists. 
+
+            if target_path == "/":
+                source.copy(h5_path, "/") if source == target else source.copy(h5_path, target)
+
```

Comment:
Sorry, my earlier comment was unclear. What I find helpful is the following:

```python
if x:
    foo()
elif y:
    bar()
else:
    baz()

if z:
    boa()
else:
    ranoutofsounds()
```

So that independent clauses have space between them

Reply:
Done